release-notes/app-catalog/app-platform-announcement.md

```diff
@@ -0,0 +1,46 @@
+# Announcing Giant Swarm App Platform
+We are pleased to announce that the Giant Swarm App Platform is generally available.
+
+In the past few months, you may have noticed that we have been working on this. We even answered some questions we received as a result of the preview. This has helped us shape the platform and bring it to where it is today.
+
+The App Platform was designed to support multiple App Catalogs. Let’s take a look at some examples:
```

Comment:
```suggestion
There are two ways to benefit:

* One, install the apps in the Giant Swarm Catalog on your clusters. Some apps we are working towards fully supporting are NGINX Ingress Controller, Kong, and Prometheus Operator.
* Two, build your own App Catalogs and build a pipeline of your own apps into your clusters.

Below, you can find more details on the multiple catalogs in the current App Platform.
```

In this [comment](https://github.com/giantswarm/giantswarm/issues/8444#issuecomment-619953771), @teemow suggested more specific benefits. I agree this wording helps the customer better imagine how they might benefit.

Reply:
WDYT @Oshratn , @puja108 ?

Reply:
Similar to posting release notes in Slack, I want to use the summary paragraph in Slack like below.

---

# Announcing Giant Swarm App Platform
We are pleased to announce that the Giant Swarm App Platform is generally available.

In the past few months, you may have noticed that we have been working on this. We even answered some questions we received as a result of the preview. This has helped us shape the platform and bring it to where it is today.

There are two ways to benefit:

* One, install the apps in the Giant Swarm Catalog on your clusters. Some apps we are working towards fully supporting are NGINX Ingress Controller, Kong, and Prometheus Operator.
* Two, build your own App Catalogs and build a pipeline of your own apps into your clusters.

Find more details on the multiple catalogs in the current App Platform here: https://github.com/giantswarm/releases/blob/announce-app-platform/release-notes/app-catalog/app-platform-announcement.md

Reply:
LGTM!